grpc/src/compiler/cpp_generator.cc

```diff
@@ -178,10 +186,11 @@ grpc::string GetHeaderIncludes(grpc_generator::File *file,
 
 void PrintHeaderClientMethodInterfaces(
     grpc_generator::Printer *printer, const grpc_generator::Method *method,
+    const grpc_cpp_generator::Parameters &params,
     std::map<grpc::string, grpc::string> *vars, bool is_public) {
   (*vars)["Method"] = method->name();
-  (*vars)["Request"] = method->input_type_name();
-  (*vars)["Response"] = method->output_type_name();
+  (*vars)["Request"] = GetCppClassName(params, method->get_input_type());
```

Comment:
These *._generator.cc files are copied from the grpc project, so we should avoid making changes in them, as this requires more difficult merging later.